salt/elife-bot/init.sls

```diff
@@ -115,22 +115,15 @@ elife-bot-digest-cfg:
 #
 
 elife-bot-temporary-files-cleaner:
-    file.managed:
+    file.absent:
         - name: /opt/rmrf_enter/elife-bot.py
-        - source: salt://elife-bot/config/opt-rmrf_enter-elife-bot.py
-        - template: jinja
-        - makedirs: True
-        - require:
-            - global-python-requisites
 
     # 2am, every day
     cron.present:
         - identifier: temp-files-cleaner
-        - name: python /opt/rmrf_enter/elife-bot.py > /dev/null
+        - name: find . -maxdepth 1 -type d -name '20*' -mtime +{{ pillar.elife_bot.rmrf_enter.days }} -exec rm -r '{}' \; >> /tmp/elife-bot-temporary-files-cleaner.log
```

Comment:
Please excuse if I don't fully understand how `rmft_enter` was used, but here I assume the `find .` this is already in the `/opt/elife-bot/tmp` or `/bot-tmp` folder by this point?

Reply:
Yeah this lacks a working directory, I'll substitute `.`

Reply:
`rmrf_enter` was/is a library that was installed on all machines and made mass deletions of files a little bit safer. It was written before bash became my go-to for this sort of thing and at the end of the day, while it worked, it wasn't maintained and it wasn't particularly worth the extra complexity that comes from maintaining a library and and per-project scripts that use that library. It's usage got whittled away to just elife-bot and now we're removing that as well.

Reply:
fixed up that working directory